griptape/drivers/image_query/anthropic_image_query_driver.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING, Optional
+from attr import define, field, Factory
+from griptape.artifacts import ImageArtifact, TextArtifact
+from griptape.drivers import BaseImageQueryDriver
+from griptape.utils import import_optional_dependency
+
+if TYPE_CHECKING:
+    from anthropic import Anthropic
+
+
+@define
+class AnthropicImageQueryDriver(BaseImageQueryDriver):
+    """
+    Attributes:
+        api_key: Anthropic API key.
+        model: Anthropic model name.
+        client: Custom `Anthropic` client.
+        max_output_tokens: Max output tokens to return.
+    """
+
+    api_key: str = field(kw_only=True, metadata={"serializable": True})
+    model: str = field(default="claude-3-sonnet-20240229", kw_only=True, metadata={"serializable": True})
+    client: Anthropic = field(
+        default=Factory(
+            lambda self: import_optional_dependency("anthropic").Anthropic(api_key=self.api_key), takes_self=True
+        ),
+        kw_only=True,
+    )
+    max_output_tokens: Optional[int] = field(default=4096, kw_only=True, metadata={"serializable": True})
+
+    def try_query(self, query: str, images: list[ImageArtifact]) -> TextArtifact:
```

Comment:
Could you normalize empty lines in this method (and other class methods)? Assignments shouldn't be separated by empty lines but groups of assignments should be separated from method calls and for/if structures for better readability.

Reply:
I'm good on most of these. I have always followed a standard though if you are initializing a variable (like an empty list) that is specifically going to be populated by a for-loop or if/else statement, then that remains "grouped" with that statement. Thoughts?

Reply:
All of these cases are no longer relevant due to changing to list comprehension, so no longer need to discuss this.